config/locales/en.yml

```diff
@@ -1840,8 +1840,18 @@ en:
     questions:
       az_charitable_contributions:
         edit:
```

Comment:
I'm assuming these were added in order to affirm that the expandable info boxes look correct